resources/languages/java/controller.i

```diff
@@ -153,8 +153,14 @@ namespace webots {
 //  Accelerometer
 //----------------------------------------------------------------------------------------------
 
+%typemap(out) const double * {
+  $result = SWIG_JavaArrayOutDouble(jenv, (double *) $1, arg1->getLookupTableSize()*3);
+}
+
 %include <webots/Accelerometer.hpp>
 
+%clear const double *;
+
```

Comment:
I will check. When I tested the API in Python it was printing only a few values, so I needed to define the list length explicitly. It makes sense because the C++ function returns only a pointer and there is no way to determine the length of the array (maybe something like https://stackoverflow.com/a/21185118/1983050).

I will do a more comprehensive verification.

Reply:
I just added `double test[3]` property to the accelerometer and it indeed returns a list of 3 values in Python. However, if I declare it as a `double test[1]` or `double test[5]` it still returns list of 3 values. From some reason 3 is a magical number for arrays, but I couldn't find why.

Reply:
I am not sure to understand, where did you add this `double test[3]`?

Reply:
Got the same behaviour for Java as well.

Reply:
> I am not sure to understand, where did you add this `double test[3]`?

I added the field (`double test[3]`), just to test the behaviour of C++/Java/Python interface.

Reply:
If you use `[X]`, this is mapped here:
https://github.com/cyberbotics/webots/blob/10b96ec42c8642cd53f7b56ee23c4d3dd7b749e1/resources/languages/java/controller.i#L71-L83

But in the simplest pointer case, it mapped to a generic `[]` here: https://github.com/cyberbotics/webots/blob/10b96ec42c8642cd53f7b56ee23c4d3dd7b749e1/resources/languages/java/controller.i#L84